PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice-java/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/webapp/action/DeployWebAppAction.java

```diff
@@ -67,11 +67,9 @@ public static void deploy(@Nonnull final Project project) {
     private static void runConfiguration(@Nonnull Project project, RunnerAndConfigurationSettings settings) {
         final RunManagerEx manager = RunManagerEx.getInstanceEx(project);
         AzureTaskManager.getInstance().runLater(() -> {
-            if (RunDialog.editConfiguration(project, settings, message("webapp.deploy.configuration.title"),
-                DefaultRunExecutor.getRunExecutorInstance())) {
+            if (RunDialog.editConfiguration(project, settings, message("webapp.deploy.configuration.title"), DefaultRunExecutor.getRunExecutorInstance())) {
```

Comment:
Will we have maven build task when create run configuration?

Reply:
I didn't verify this. the change in line 74 just set back what it already has. I believe this line won't change anything.